_episodes/format.md

```diff
@@ -88,6 +88,8 @@ Here are some recommended formats for different types:
 > Data Management Plan (DMP) covers how data can be stored, described and reused.
```

Comment:
```suggestion
> Data Management Plan (DMP) covers how data can be stored, described and reused.
> For example, see [DMPonline](https://www.dcc.ac.uk/dmponline) or [DSW](https://ds-wizard.org/).
```

Reply:
I cannot commit this somehow, but it seems like a valuable addition.

Reply:
Ah, I think it is because it is outside of the changed files. Would have to be done via separate PR I think.

Reply:
See #40.